src/resources/extensions/quarto/confluence/publish.lua

```diff
@@ -0,0 +1,283 @@
+-- Confluence Storage Format for Pandoc
+-- https://confluence.atlassian.com/doc/confluence-storage-format-790796544.html
+-- https://pandoc.org/MANUAL.html#custom-readers-and-writers
+
+local stringify = (require "pandoc.utils").stringify
+local confluence = require('overrides')
+
+-- From https://stackoverflow.com/questions/9168058/how-to-dump-a-table-to-console
+function dumpObject(o)
```

Comment:
this might be omitted as they are probably coming from `overrides` at this point?

Reply:
That, and `quarto.log.output`?